apps/schools/domains/student/components/createStudentForm/index.tsx

```diff
@@ -128,6 +160,21 @@ export const CreateStudentForm = () => {
                         placeholder='Выберите кружок'
                         className={styles.select}
                         loading={circlesData.isLoading}
+                        dropdownStyle={DROPDOWN_STYLE}
+                        dropdownRender={(menu) => (
+                            <div className={styles.dropdown}>
+                                {menu}
+                                <Button
+                                    type='schoolDefault'
+                                    block
+                                    style={{ width: '40%', marginLeft: '10px', marginBottom: '6px', marginTop: '10px' }}
+                                    className={styles.button}
```

Comment:
hard code styles and classname lies next to it, how bad)

Reply:
Yes, it is. Will be fixed in this PR.